projects/schematics/src/migrations/5_0/constructor-deprecations/data/cart-lib.migration.ts

```diff
@@ -0,0 +1,2253 @@
+import { ConstructorDeprecation } from '../../../../shared/utils/file-utils';
+
+export const CART_LIB_MIGRATION: ConstructorDeprecation[] = [
+  {
+    class: 'CartQuickOrderFormComponent',
+    importPath: '@spartacus/cart/quick-order/components',
+    deprecatedParams: [
+      {
+        className: 'ActiveCartService',
+        importPath: '@spartacus/core',
+      },
+      {
+        className: 'EventService',
+        importPath: '@spartacus/core',
+      },
+      {
+        className: 'FormBuilder',
+        importPath: '@angular/forms',
+      },
+      {
+        className: 'GlobalMessageService',
+        importPath: '@spartacus/core',
+      },
+    ],
+    removeParams: [
+      {
+        className: 'ActiveCartService',
+        importPath: '@spartacus/core',
+      },
+    ],
+    addParams: [
+      {
+        className: 'ActiveCartFacade',
+        importPath: '@spartacus/cart/base/root',
+      },
+    ],
+  },
+  {
+    class: 'QuickOrderComponent',
+    importPath: '@spartacus/cart/quick-order/components',
+    deprecatedParams: [
+      {
+        className: 'ActiveCartService',
+        importPath: '@spartacus/core',
+      },
+      {
+        className: 'CmsComponentData',
+        importPath: '@spartacus/storefront',
+      },
+      {
+        className: 'GlobalMessageService',
+        importPath: '@spartacus/core',
+      },
+      {
+        className: 'QuickOrderFacade',
+        importPath: '@spartacus/cart/quick-order/root',
+      },
+      {
+        className: 'QuickOrderStatePersistenceService',
+        importPath: '@spartacus/cart/quick-order/core',
+      },
+    ],
+    removeParams: [
+      {
+        className: 'ActiveCartService',
+        importPath: '@spartacus/core',
+      },
+    ],
+    addParams: [
+      {
+        className: 'ActiveCartFacade',
+        importPath: '@spartacus/cart/base/root',
+      },
+    ],
+  },
+  {
+    class: 'QuickOrderService',
```

Comment:
there is `quick-order-form.component.migration.ts` inside `data` folder. There are some differences between this one and that (they should be merged, not just remove one of them).